src/website/views/components/VoyagePageTemplate/VoyagePageTemplate.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './VoyagePageStyles.css'; 
+
+function VoyagePageTemplate({ desc }) {
```

Comment:
Sorry I may have missed it, but is this component being used anywhere?

Reply:
Oops, that component is not being used anymore. I was using it before, but forgot to remove it after I decided not to use it.

Reply:
Ok all good